### PR TITLE
Fixed bugs of S3 and create an example of tensorflow framework in Federated Learning

### PR DESCRIPTION
**Pull request by XinYao1994**
This is a PR for fixing some bugs in the PR of integrating Plato into Sedna to support federated learning (#116).

* We fix some S3 bugs in ```Sedna v0.4.0 ```.
* We add one example of using the TensorFlow framework with the new Sedna interface.

### CODE REVIEW

**Comment by JoeyHwong-gk** [comment]
Please add new example into  `docs/index.rst`

File: examples/federated-learning-surface-defect-detection-aggregation.Dockerfile [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+
 RUN pip install -r /home/requirements.txt
+RUN pip install keras
+RUN pip install tensorflow-datasets
 
-ENV PYTHONPATH "/home/lib"
+ENV PYTHONPATH "/home/lib:/home/plato"
 
 COPY ./lib /home/lib
+RUN git clone https://github.com/TL-System/plato.git /home/plato
```
Comment (JoeyHwong-gk) [inline examples/federated-learning-surface-defect-detection-aggregation.Dockerfile:17]: Doesn't seem to need `package` in `plato`. suggest `pip install plato-learn`

File: examples/federated-learning-surface-defect-detection-train.Dockerfile [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
 
 COPY ./lib/requirements.txt /home
 RUN pip install -r /home/requirements.txt
-RUN pip install keras~=2.4.3
-RUN pip install opencv-python==4.4.0.44
-RUN pip install Pillow==8.0.1
+RUN pip install keras
+RUN pip install opencv-python
+RUN pip install Pillow
+RUN pip install tensorflow-datasets
```
Comment (JoeyHwong-gk) [inline examples/federated-learning-surface-defect-detection-train.Dockerfile:15]: Where was `tensorflow-datasets` used?

Comment (XinYao1994) [reply examples/federated-learning-surface-defect-detection-train.Dockerfile:15]: This is used by Plato build-in dataset

File: examples/federated-learning-surface-defect-detection-aggregation.Dockerfile (hunk 1 above)
Comment (XinYao1994) [reply examples/federated-learning-surface-defect-detection-aggregation.Dockerfile:17]: I will update it until ```Plato``` becomes stable.

**Comment by llhuii** [comment]
/lgtm

> Basically look good to me, thanks. Leave the decision to @jaypume @JoeyHwong-gk @JimmyYang20

**Pull request merged**